src/components/KanaTable/BasicKanaTable.tsx

```diff
@@ -0,0 +1,76 @@
+import KanaType from '../../types/kana';
+import { Caption, Table } from './KanaTableStyles';
+import TableHead from './TableHead';
+import TableRow from './TableRow';
+
+type BasicKanaTableProps = {
+  kana: KanaType[];
+  caption: string;
+};
+
+export default function BasicKanaTable({ kana, caption }: BasicKanaTableProps) {
```

Comment:
If you like, you can add a return type for your component
```suggestion
export default function BasicKanaTable({ kana, caption }: BasicKanaTableProps): JSX.Element {
```

Reply:
Thank you for your review! 🙌  I am still a bit unfamiliar with TypeScript and was wondering whether it would be best practice to add a type for the return value of my component. What do you think?

Reply:
I think that it is always good to specify a return type that you expect from a function. When declaring variables it is not always necessary to specify a type and can be derived from typescript as well, same for functions. But as I said the return type is not wrong, so that you also see at their places in the code when using a function what you get back from the function.